### ISSUE TITLE
Suggestion for new package covering missing stat functions

### ISSUE THREAD

**Issue by msberends** [easystats/easystats#144]
Would it be an idea to add another package with stat functions you would expect to be in `stats` but they are not?

Harmonic mean, geometric mean, mode, correlation of variance, standard errors, these kind of things? I could things of tens of obvious functions.

**Comment by bwiernik** [comment]
I'm not sure exactly what you mean by correlation of variance or standard errors. Could you elaborate?

**Comment by msberends** [comment]
As far as I know, there is no `cv()` function such as:

```r
# coefficient of variation
cv <- function(..., na.rm = FALSE) {
  sd(c(...), na.rm = na.rm) / mean(..., na.rm = na.rm)
}
```

Another example, the coefficient of quartile variation (CQV, [example in other package](https://cran.r-project.org/web/packages/cvcqv/vignettes/cqv_versatile.html)):

```r
cqv <- function(x, na.rm = FALSE) {
  q1 <- quantile(x, 0.25)
  q3 <- quantile(x, 0.75)
  (q3 - q1) / (q3 + q1)
}
```

Wouldn’t the easystats be a great place for missing, obvious stat functions?

**Comment by msberends** [comment]
Ah, sorry, I now see I wrote ‘correlation’, where I meant ‘coefficient’. Excuse me!

**Comment by DominiqueMakowski** [comment]
I don't think there's enough of these for now to warrant their own separate package, but they could probably be incorporated within others, such as for instance bayestestR or parameters that have some functions to describe vectors

**Comment by bwiernik** [comment]
I think these could go in either parameters or datawizard

**Comment by IndrajeetPatil** [comment]
Yes, this would be a nice addition. I would vote for including them in `datawizard`.

**Comment by strengejacke** [comment]
CV (currently also in [sjstats](https://strengejacke.github.io/sjstats/reference/cv.html)) is something that would fit into _performance_, since it somewhat close to r2() or rmse().

**Comment by strengejacke** [comment]
Maybe also some of the [summary statistics](https://strengejacke.github.io/sjstats/reference/index.html#section-other-summary-statistics) from sjstats can be implemented in datawizard?

**Comment by msberends** [comment]
> I don't think there's enough of these for now to warrant their own separate package

This could use some investigation. I’d like to think the opposite, but I’ll look around for blogs and question of people missing such functions in base R or ‘stats’.

**Comment by IndrajeetPatil** [comment]
> Maybe also some of the summary statistics from sjstats can be implemented in datawizard?

Yes, a few of them can definitely be implemented.

Also, the weighted functions you have might also be helpful to be included in `datawizard`.

**Comment by bwiernik** [comment]
The describe distribution functions in general could be moved from parameters to datawizard (and re-exported).
